docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1695,6 +1695,15 @@ Note: If you would like to move
 
 - This HOC was removed. There's an alternative using the `useMediaQuery` hook on [this page](/components/use-media-query/#migrating-from-withwidth).
 
+#### experimentalStyled
```

Comment:
I`

Reply:
I saw on one of the recent issues people complaining about not documenting some BC on the new utils. We could only mention it on the changelog though.. Would that work better?

Reply:
I'm asking because v5 seems to have 1% of usage compared to v4. Developers using v4, the targeted audience of this page, and seeing this could be confused, e.g. what is this module?

Reply:
Let’s include then only on the changelog.